<launch>
   <!-- <include file="$(find lio_sam)/launch/run.launch"/> -->
  <node pkg="imupoint" type="pcd2topic" name="pcd2topic" output="screen" />

  <node pkg="octomap_server" type="octomap_server_node" name="octomap_server">
        <param name="resolution" value="0.05" />
        <param name="frame_id" type="string" value="map" />
        <param name="sensor_model/max_range"  value="1000.0" />
        <param name="pointcloud_max_z" value="0.2"/>
        <param name="pointcloud_min_z" value="-0.2"/>
        <param name="height_map" value="false"/>
        <param name = "colored_map" value = "true" /> 
        <remap from="cloud_in" to="/cloud_map"/>
        <!-- <remap from="cloud_in" to="/cloud_map_old"/> -->
<!-- /lio_sam/mapping/cloud_registered -->
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find imupoint)/rviz/grid.rviz" />
</launch>